misc/workflow-with-failures/env0.workflow.yml

```diff
@@ -0,0 +1,16 @@
+environments:
+  A:
+    name: A
+    templateName: "null"
+  B:
+    name: B
+    templateName: "null switch"
```

Comment:
what is this `null switch`?

Reply:
The [null switch template](https://dev.dev.env0.com/templates/622c8542-6360-4aec-ae05-20093c4dc5c0/settings). it can simulate a failure using the `SHOULD_PASS` env var.